moses/moses/main/populate_atomspace.cc

```diff
@@ -0,0 +1,72 @@
+#include <moses/data/table/table.h>
+#include <moses/data/table/table_io.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <moses/utils/value_key.h>
+#include <opencog/atoms/base/Link.h>
+#include "populate_atomspace.h"
+#include <opencog/atoms/value/Value.h>
+
+namespace opencog {
+	namespace moses {
+
+		void populate(AtomSpace *as, ITable &itable) {
```

Comment:
@Eman22S according to the opencog coding standard https://wiki.opencog.org/w/Development_standards#Formatting you can ignore the indentation of namespace blocks, which I would recommend since you lines are moderately long.  Meaning that you can remove 2 levels of indentations (one from opencog, one from moses). So for instance
```c++
void populate(AtomSpace *as, ITable &itable) {
```
would start right at indentation level zero.

Reply:
so should `namespace moses ` be on line 9 and the void function on line 10?

Reply:
@Eman22S `namespace moses` could be on line 9 or 10 as you please. I think `void populate(...` should have a new line before it (thus line 11 or 12).